packages/sil-governance/src/routes/create-proposal/components/ProposalForm.tsx

```diff
@@ -48,11 +51,30 @@ const ProposalForm = (): JSX.Element => {
       electionRuleId,
     };
 
-    const text = values[TEXT_FIELD];
-
     switch (proposalType) {
-      case ProposalType.AmendProtocol:
+      case ProposalType.AddCommitteeMember: {
+        const committee = parseInt(
+          values[COMMITTEE_ADD_FIELD].substring(0, values[COMMITTEE_ADD_FIELD].indexOf(":")),
+          10,
+        ) as CommitteeId;
```

Comment:
Can you extract this into a `getCommunityIdFromForm(values[COMMITTEE_ADD_FIELD])` helper?